cpp/tests/utilities/column_utilities.cu

```diff
@@ -371,5 +372,28 @@ void print(cudf::column_view const& col, std::ostream &os, std::string const& de
   os << to_string(col, delimiter);
 }
 
+bool validate_host_masks(std::vector<bitmask_type> const& expected_mask,
+                         std::vector<bitmask_type> const& got_mask,
+                         size_type number_of_elements) {
+
+    std::vector<cudf::size_type> mask_index(expected_mask.size());
+    std::iota(mask_index.begin(), mask_index.end(), 0);
+    size_type bitmask_size = sizeof(bitmask_type) * 8;
+
+    return std::all_of(mask_index.begin(), mask_index.end(),
+            [&expected_mask, &got_mask, number_of_elements, bitmask_size](auto index){
+                if ((index+1)*bitmask_size <= number_of_elements){
```

Comment:
Since this is just test code and we don't really care that much about performance, I think it would be easier to read this code by just testing individual bits using `bit_is_set` https://github.com/rapidsai/cudf/blob/branch-0.13/cpp/include/cudf/utilities/bit.hpp#L91